include/exadg/incompressible_navier_stokes/postprocessor/output_generator.h

```diff
@@ -77,6 +78,9 @@ struct OutputData : public OutputDataBase
   // write vorticity magnitude
   bool write_vorticity_magnitude;
 
+  // write wall shear stress on IDs
+  std::vector<dealii::types::boundary_id> write_wall_shear_stress_on_IDs;
```

Comment:
I think the name should include "boundary", e.g. `_boundary_IDs` or short `_bids`.

Reply:
I did, good point.